src/js/covidMap.js

```diff
@@ -420,34 +499,78 @@ fluid.defaults("fluid.covidMap.map", {
                 }
             }
         },
-        selectedHospitalPane: {
-            type: "fluid.covidMap.hospitalRenderer",
-            container: "{that}.dom.hospitalPanel",
+        filterCountOnDesktop: {
+            type: "fluid.covidMap.filterCount",
+            container: "{that}.dom.filterCountOnDesktop"
+        },
+        filterCountOnMobile: {
+            type: "fluid.covidMap.filterCount",
+            container: "{map}.dom.filterCountOnMobile"
+        },
+        locationButtonOnMobile: {
+            type: "fluid.button",
+            container: "{map}.dom.locationButtonOnMobile",
             options: {
-                gradeNames: "fluid.viewComponent",
-                model: {
-                    row: "{map}.model.selectedHospital"
-                },
-                selectors: {
-                    hospitalWebsite: ".fl-mapviz-hospital-website" // This field left over from fluid.covidMap.hospitalRenderer
+                modelListeners: {
+                    "resetQuery": {
+                        path: "activate",
+                        changePath: "{map}.model.query",
+                        value: ""
+                    }
                 },
                 modelRelay: {
-                    hospitalWebsite: {
-                        source: {
-                            segs: ["row", "{map}.options.fields.website"]
-                        },
-                        target: "dom.hospitalWebsite.text"
+                    showCitiesList: {
+                        source: "activate",
+                        target: "{map}.model.visiblePanelOnMobileFlags",
+                        func: x => {
+                            if (x > 0) {
+                                return {
+                                    "citiesList": true,
+                                    "resultsPage": false,
+                                    "filterPanel": false,
+                                    "hospitalPanel": false
+                                };
+                            };
+                        }
+                    },
+                    expandCitiesList: {
+                        source: "activate",
```

Comment:
As below this is better as
{code}
    modelListeners: {
        expandCitiesList: {
            path: "activate",
            changePath: "{citiesList}.expandButton.model.expanded",
            value: true,
            excludeSource: "init"
        }